sernet.gs.ui.rcp.main/src/sernet/gs/ui/rcp/main/common/model/CnAElementHome.java

```diff
@@ -255,11 +255,19 @@ public CnALink createLink(CnATreeElement dropTarget, CnATreeElement dragged, Str
     public void remove(CnATreeElement element) throws CommandException {
         if (log.isDebugEnabled()) {
             log.debug("Deleting element, uuid: " + element.getUuid()); //$NON-NLS-1$
+            log.debug("Are Module templates active?: " + isModelingTemplateActive()); //$NON-NLS-1$
+        }
+
+        if (isModelingTemplateActive()) {
```

Comment:
The way I see it you do exactly the same thing with both commands, i.e. you can move most of the code within the if scope:
```java
if (...) {
  command = new ..
} else {
  command = new ...
}
delete...
...
```

Reply:
done